### ISSUE TITLE
Mapbox Telemetry should be hidden in attribution action sheet if MGLMapboxAccessToken is not set

### ISSUE THREAD

**Issue by PrestonL** [mapbox/mapbox-gl-native#8469]
**Platform:** iOS
**SDK Version:** 3.4.7

### Steps to trigger behavior

 1. Do not set MGLMapboxAccessToken
 2. Whenever clicking the Attribution "i", the "Mapbox Telemetry" box is present, even though no telemetry will be sent.

### Expected behavior

1. Developer should be able to either remove the "Mapbox Telemetry" item from the attribution sheet
-- or --
2. MapBox SDK should remove the "Mapbox Telemetry" option whenever MGLMapboxAccessToken is not set

**Comment by 1ec5** [comment]
Thanks for the suggestion. You’re right that the Mapbox Telemetry item is unnecessary in this case. It’ll be easier to fix this issue (or work around it) once #8373 lands.

**Comment by pimnijman** [comment]
What's the status on this? I would like to remove the telemetry option from the action sheet.

**Comment by alfwatt** [comment]
@friedbunny what is the use case for Maps SDK without an access token? Will the Map view be able to load tokens?

**Comment by friedbunny** [comment]
@alfwatt Yes, the iOS Maps SDK supports the use of third-party resources without an access token.
